cli/index.ts

```diff
@@ -0,0 +1,43 @@
+// Use commander or whatever to start CLI
+// Make it use dev mode for devserver, make it use built next app for regular usage
+// aiconfig devserver
+// aiconfig server
```

Comment:
nit, maybe a more distinct command for the regular usage case -- `aiconfig editor` for example?

Reply:
This can get updated in README once we're publishing, right now just have the dev loop written in the CLI readme